src/Search/Gearman/Command/ImportCommand.php

```diff
@@ -157,6 +167,12 @@ private function iterateSerializeTask(Iterator $items, string $type, $method = '
                     $items->next();
                     continue;
                 }
+
+                if (!is_null($this->dateFrom) && ($item->getPublishedDate() < $this->dateFrom)) {
```

Comment:
We could use `getStatusDate()` if `$item instanceof ArticleVersion` or `$item instanceof ReviewedPreprint` and then we should check if `$item instanceof HasPublishedDate` before using `getPublishedDate()`.

If there is neither `getStatusDate()` or `getPublishedDate` then we should not index that item if the dateFrom option is set. But I don't think we have any content that will have neither.

As this condition and the previous one have the same consequence we could just compound the logic.